src/api/ContextMenu.ts

```diff
@@ -89,15 +89,18 @@ export function removeGlobalContextMenuPatch(patch: GlobalContextMenuPatchCallba
 }
 
 /**
- * A helper function for finding the children array of a group nested inside a context menu based on the id of one of its childs
- * @param id The id of the child
+ * A helper function for finding the children array of a group nested inside a context menu based on the id(s) of its children
+ * @param id The id(s) of the child/children
```

Comment:
this doc is kinda misleading, it makes it sound like all children must be present
```suggestion
 * @param id The id of the child. If an array is specified, all ids will be tried
```